data/gala.gschema.xml

```diff
@@ -7,11 +7,10 @@
     <value nick="minimize-current" value="3" />
     <value nick="open-launcher" value="4" />
     <value nick="custom-command" value="5" />
-    <value nick="window-overview" value="6" />
-    <value nick="window-overview-all" value="7" />
-    <value nick="switch-to-workspace-previous" value="8" />
-    <value nick="switch-to-workspace-next" value="9" />
-    <value nick="switch-to-workspace-last" value="10" />
+    <value nick="window-overview-all" value="6" />
```

Comment:
I wonder if changing the value here will mess up system settings? It seems odd that the value is a number here tbh. But might need to preserve the numbers for now even though there would be a missing option just to not break things

Reply:
I think this enum is used only in hotcorners key, and settings plug set the value using string and not actual enum value. So it shouldn't break anything.